docs/scenarios/stacks.feature

```diff
@@ -0,0 +1,138 @@
+Feature: As a library member of staff I want to login so I can place a request
+  # Note: users can currently login with either their Wellcome Trust account email address, library card number or library username
+
+  Scenario: User supplies correct user name and password
+
+    Given that I am on the login page
+    When I enter my user name and password correctly and click ‘Login’
+    Then I am taken to the xxxx page
+
+  Scenario: User does NOT enter correct user name and password
+
+    Given that I am on the login page
+    When I enter my user name and password incorrectly and click ‘Login’
+    Then I see an error message ‘incorrect username or password’
+
+
+Feature: As a library member of staff I want to know if an item is available for requesting
+
+  Scenario: A user searches for a work and it’s available to request
+
+    Given that I have searched for an item
+    When I click onto the work page I’m interested in
+    Then I can see the work is available to request
+    And I can see the location of the item
+    And I can see a ‘request it’ button
```

Comment:
I might avoid this interface specific and fold it into `Then I can see the work is available to request` where the "request it button" affordance is implicit.

In general with scenarios I feel it's better to describe the required outcome rather than the specific implementation detail (which can be hashed out in more detailed designs).